### ISSUE TITLE
Can't connect to mempool signet server ssl://mempool.space:60602

### ISSUE THREAD

**Issue by LLFourn** [bitcoindevkit/rust-electrum-client#152]
It has a valid certificate. The error I get something like this:

```
Made one or multiple attempts, all errored:\n\t- received fatal alert: DecodeError
```

**Comment by oleonardolima** [comment]
It should've been fixed by #150. Are you still facing this error?

**Comment by LLFourn** [comment]
Oh thanks @oleonardolima indeed I was on an old version. I'll assume it's fixed and re-open if not.
